### ISSUE TITLE
Getting collect2.exe: error: ld returned 1 exit status when compiling

### ISSUE THREAD

**Issue by levonmaister** [MJoergen/mchess#1]
Can't get it up running. Been compiling it with g++ and it won't work.

**Comment by MJoergen** [comment]
Hmm. I just tried right now:

1. Clone repository
2. type `make`

and that worked fine.

I'm running Linux Mint (variant of Ubuntu). I guess you're on a Windows machine. Are you using cygwin ?

**Comment by levonmaister** [comment]
Yeah I got it working on Linux. But haven't tried since on Windows.
